apps/photos/src/components/Collections/CollectionShare/emailShare.tsx

```diff
@@ -2,17 +2,49 @@ import SingleInputForm, {
     SingleInputFormProps,
 } from 'components/SingleInputForm';
 import { GalleryContext } from 'pages/gallery';
-import React, { useContext } from 'react';
+import React, { useContext, useState, useEffect } from 'react';
 import { t } from 'i18next';
 import { shareCollection } from 'services/collectionService';
 import { User } from 'types/user';
 import { handleSharingErrors } from 'utils/error/ui';
 import { getData, LS_KEYS } from 'utils/storage/localStorage';
 import { CollectionShareSharees } from './sharees';
+import { getLocalCollections } from 'services/collectionService';
 
 export default function EmailShare({ collection }) {
     const galleryContext = useContext(GalleryContext);
 
+    const [updatedOptionsList, setUpdatedOptionsList] = useState(['']);
+    let updatedList = [];
+    useEffect(() => {
+        const owner_user1: User = getData(LS_KEYS.USER);
```

Comment:
so we normally use `camelCasing` with variable name

So lets call this `ownerUser`

Reply:
Variable name changes to ownerUser